### ISSUE TITLE
Why, and how?

### ISSUE THREAD

**Issue by vidluther** [enygma/php53-examples#1]
I think these examples are great, but if you could add examples of why one would want to use Closures, or the new OOP stuff would be awesome. A lot of documentation and posts about 5.3 talks to a crowd that's already in the know, but for the non serious programmer, a real world story of when a closure would be better than a standard function, would be awesome. 

PS: Yes, this is so I can understand why I should be excited about all these new features, except Namespaces :).

**Comment by enygma** [comment]
Good idea...I think once I get the simple examples of the features in there, I might make "stories" that use them too. This is for my education too :)

**Comment by vidluther** [comment]
eggcellent
